### ISSUE TITLE
A11y_WINFORMS_CommonControls2_MonthCalendar_Keyboard : Keyboard focus order is incorrect inside "Common Control 2" section

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/winforms#7337]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### .NET version

.NET Framework Version 4.8.04161
Operating System: Windows 11 Enterprise 21H2

### Did it work in .NET Framework?

No

### Issue description

GitHubTags:#A11yMAS;#A11yTCS;#.NET Core[WINFORMS]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG2.4.3;#DesktopApp;#Win11;#Benchmark;#Keyboard

### Actual:
As soon as the "Common Control 2" dialog gets open keyboard focus is first moving to "Web Browser" controls which is incorrect.

### Expected:
As soon as the "Common Control 2" dialog gets open keyboard focus should first move to "Month Calendar" controls and then to others control like "Numericc UpDown, Radio button" like wise keyboard order should be there in the dialog.

### User Impact:
Keyboard users will face difficulties while navigating common control 2 dialog.

### Steps to reproduce

1. Open Core testing solution in VS Enterprise 2022 Int Preview.
2. Right click on cs.proj file and click on Build.
3. Press enter on the application.
4. Main Form screen would appear.
5. Click on "Common_Controls2:Testing the controls under the Common_controls" Tab.
6. Tab to MonthCalendar Section.
7. Navigate using Tab key and observe that Keyboard focus order is incorrect inside "Common Control 2" section

### Attachment:
[7337_A11y_WINFORMS_CommonControls2_MonthCalendar_Keyboard MP4.zip](https://github.com/dotnet/winforms/files/8974801/7337_A11y_WINFORMS_CommonControls2_MonthCalendar_Keyboard.MP4.zip)

**Comment by merriemcgaw** [comment]
@PoojaNamde could you please attach a video explaining what is wrong with the tab order for us? This is not enough information.

**Comment by PoojaNamde** [comment]
@merriemcgaw 

Added attachment describing the bug.

**Comment by merriemcgaw** [comment]
Thanks @PoojaNamde .

@Olina-Zhang can you replace the website content with the simple html doc below (you'll need to change the extension I think). With very little content I'm interested in finding out where the focus goes next. Just attach the updated sample and assign back to me. Thanks!

[simple.txt](https://github.com/dotnet/winforms/files/8980638/simple.txt)

**Comment by Amy-Li03** [comment]
Thank you @merriemcgaw and @Tanya-Solyanik , it works to add a simple html for webBrowser, have updated the app:
[Accessibility_Core_7.0_App.zip](https://github.com/dotnet/winforms/files/8989207/Accessibility_Core_7.0_App.zip), we will update the Accessibility app in GitHub later.
@PoojaNamde please use the new app to do testing.

**Comment by PoojaNamde** [comment]
Verified this issue in the latest environment. Issue is fixed hence closing the bug.

**Comment by PoojaNamde** [comment]
Verified this issue in the latest environment. Issue is fixed hence closing the bug.

**Comment by MandiMan** [comment]
Verified on .NET 7.0.100-preview.7.22370.3 TP build, issue is fixed, screenshot as below:
![issue7359](https://user-images.githubusercontent.com/108860782/180945038-5a8493bf-4f02-46c7-9b52-6eb4dcd2f8ee.gif)
